tests/test_unittest/test_gtc/gtir_utils.py

```diff
@@ -0,0 +1,20 @@
+from gt4py.gtc.gtir import CartesianOffset, FieldAccess, TernaryOp, Expr
+from gt4py.gtc.common import DataType
+
+
+class DummyExpr(Expr):
+    """Fake expression for cases where a concrete expression is not needed."""
+
+    dtype: DataType = DataType.FLOAT32
+
+
+class FieldAccessBuilder:
+    def __init__(self, name) -> None:
```

Comment:
I used name as required because I thought having a default name might cause problems (if you accidentally use the same name twice).